index.bs

```diff
@@ -344,6 +346,9 @@ The <dfn export>low entropy hint table</dfn> below defines hints that are only e
  <tr>
   <td>`Sec-CH-UA-Platform`
   <td>a suitable <a href=https://wicg.github.io/ua-client-hints/#sec-ch-ua-platform>Platform value</a>
+ <tr>
+  <td>`Sec-CH-Prefers-Color-Scheme`
```

Comment:
Yeah, can we remove this? This implies that we send it by default for _all_ requests.

Reply:
(Note: we should make that more obvious in the spec, see https://github.com/WICG/ua-client-hints/issues/225 - which is maybe in the wrong repo :shrug:)

Reply:
Removed.